packages/tame-metering/test/test-sanity.js

```diff
@@ -0,0 +1,17 @@
+import '../src/install-global-metering';
+import test from 'tape-promise/tape';
+
+test('symbol properties', t => {
+  t.assert(RegExp[Symbol.species], 'RegExp[Symbol.species] is kept');
+  t.end();
+});
+
+function foo(_bar) {
+  // eslint-disable-next-line no-eval
+  return eval('_bar');
```

Comment:
Very illustrative test case. Since this is just one sane expectation, should the test file should be named `direct-eval.test.js`?

Reply:
The symbol properties test is also in this file.